kytos/core/auth.py

```diff
@@ -290,7 +290,7 @@ def _authenticate_user(self):
             token = self._generate_token(username, time_exp)
             return {"token": token}, HTTPStatus.OK.value
         except Unauthorized:
-            result = "Incorrect username or password"
+            result = "Incorrect password"
```

Comment:
Change the error message because `Unauthorized` only happens when `password` is incorrect. Incorrect `username` raises `NotFound`.

Reply:
Sounds good.